core/src/main/java/org/jdbi/v3/core/statement/BaseStatement.java

```diff
@@ -24,26 +24,22 @@
 import org.jdbi.v3.core.config.Configurable;
 import org.jdbi.v3.core.generic.GenericTypes;
 
-abstract class BaseStatement<This> implements Closeable, Configurable<This>
-{
+abstract class BaseStatement<This> implements Closeable, Configurable<This> {
     @SuppressWarnings("unchecked")
     final This typedThis = (This) this;
 
     private final Handle handle;
     private final StatementContext ctx;
     private final Collection<StatementCustomizer> customizers = new ArrayList<>();
 
-    BaseStatement(Handle handle)
-    {
+    BaseStatement(Handle handle) {
         this.handle = handle;
         this.ctx = new StatementContext(
                 handle.getConfig().createCopy(), handle.getExtensionMethod());
-    }
-
-    {
+    } {
```

Comment:
Why don't we just fold this initializer into the constructor? i.e. remove line 39

Reply:
done